mne/viz/topo.py

```diff
@@ -457,7 +457,7 @@ def _imshow_tfr(
         else:
             cbar = plt.colorbar(mappable=img, ax=ax)
         if interactive_cmap:
-            ax.CB = DraggableColorbar(cbar, img)
+            ax.CB = DraggableColorbar(cbar, img, kind=None, ch_type=None)
```

Comment:
```suggestion
            ax.CB = DraggableColorbar(cbar, img, kind="tfr_image", ch_type=None)
```
can we give a `kind` here? something like this?